src/test/binaryutils_test.cpp

```diff
@@ -420,6 +420,371 @@ TEST_F(BinaryUtilsTest, positive_range_resolution_matching_in_between) {
     }
 }
 
+class BinaryUtilsTestKFS : public ::testing::Test {
+protected:
+    void SetUp() override {
+        readRgbJpg(filesize, image_bytes);
+        inferTensorContent.mutable_contents()->add_bytes_contents(image_bytes.get(), filesize);
+    }
+
+    size_t filesize;
+    std::unique_ptr<char[]> image_bytes;
+    ::inference::ModelInferRequest::InferInputTensor inferTensorContent;
+};
```

Comment:
Most if not all of those tests are repeated from TFS binary inputs tests. Could we avoid copying & just refactor it to have template helper methods?